src/lib/components/workflow/workflow-count.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+  import type { WorkflowStatus as Status } from '$lib/types/workflows';
+  import WorkflowStatus from '../workflow-status.svelte';
+
+  export let status: Status;
+  export let count: number = Math.round(Math.random() * 10000);
+  export let active = false;
+  export let onStatusClick: (status: Status) => void = () => {};
+</script>
+
+<div
```

Comment:
There are responsive designs for these, will send you a link! 

![Screenshot 2023-08-08 at 4 24 20 PM](https://github.com/temporalio/ui/assets/15069288/82281d51-7ada-46e9-98ce-5d4e53cb5fdf)